internal/featuretests/replaceprefix_test.go

```diff
@@ -110,6 +114,9 @@ func basic(t *testing.T) {
 			envoy.RouteConfiguration("ingress_https"),
 		),
 		TypeUrl: routeType,
+	}).Status(vhost).Equals(projcontour.Status{
+		CurrentStatus: dag.StatusInvalid,
```

Comment:
its kinda gross we have to couple the feature test package to the dag package just for this constant. What happens if you use the string "invalid" here. Yeah, that's duplicating information, but that might be worth it for the decoupling, currently the feature tests know nothing about the dag.

Reply:
@davecheney I think the right fix for this is to move those constants into the `k8s` package. WDYT?

Reply:
there are only three states "valid", "invalid" and "orphaned" all of which 

a. don't change
b. are shorter than their respective declarations

It seems simpler to just use the literal constant and avoid the import depenedency.

Reply:
Maybe we should get rid fo the constant then?

Reply:
SGTM 👍 👍

Reply:
The benefit of a constant is that you can find where it is used (and also improve type checking).

Reply:
The downside is it creates strong source coupling between packages that shouldn't know anything about each other. 1/2 a dozen of one, six of the other.

Reply:
In that case, I'd prefer to move it to the k8s package.

Reply:
Sounds good to me!